synapse/models/inet.py

```diff
@@ -3749,6 +3778,16 @@ def getModelDefs(self):
                             'doc': 'The file which was attached to the message.'}),
                     )),
 
+                    ('inet:service:emote', {}, (
```

Comment:
Is the a such thing as an 'unemote'? For example, when you :heart: something, and then remote it because it was an accident, or you meant to send a different emoji.